### PR TITLE
New Voice command search bar added

### LINKED ISSUE #242: [Feature] New Voice command search bar component
### Description

I will add a new **Voice command search bar** in Search bar component.

I will use **HTML, CSS** and **Javascript** for this.

Feature of voice command will added using **javascript (web speech API).**

Kindly assign this to me under IWOC2024.

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

### PR DESCRIPTION
**Pull request by Prince-kushwaha-2004**
# Fixes Issue🛠️

#242  New Voice command search bar component added

Closes #242 

# Description👨‍💻 

I had added a new **Voice command search bar** in Search bar component.
For this I had used **HTML, CSS** and **Javascript.**
For voice command I had used web speech API.

The search bar is **fully functional,** it takes input using **voice**.
For detail kindly see attached video.

# How this has been tested✅

This has been tested by me by running the website, no error found!!

# Checklist✅ 

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have added demonstration in the form of GIF/video file
- [] I am an Open Source Contributor

# Screenshots/GIF📷

https://github.com/Rakesh9100/Beautiify/assets/149922307/70a2f726-1e93-4da8-ba6c-762a8ca7cd49

### CODE REVIEW

**Comment by Prince-kushwaha-2004** [comment]
@Rakesh9100 
sir kindly check.

File: Components/Search-Bars/Voice-Command-Search-Bar/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Rakesh9100) [inline Components/Search-Bars/Voice-Command-Search-Bar/index.html:6]: Pls change the title to your component name and always keep good title @Prince-kushwaha-2004

Comment (Prince-kushwaha-2004) [reply Components/Search-Bars/Voice-Command-Search-Bar/index.html:6]: ok i will change

**Comment by Prince-kushwaha-2004** [comment]
I had updated the title

File: Components/Search-Bars/Voice-Command-Search-Bar/index.html (hunk 1 above)
Comment (Prince-kushwaha-2004) [reply Components/Search-Bars/Voice-Command-Search-Bar/index.html]: done👍

**Comment by Rakesh9100** [comment]
Commits are fine, no worries but file changes should be of only yours, fix this @Prince-kushwaha-2004

**Comment by Prince-kushwaha-2004** [comment]
@Rakesh9100 
Deleted the extra files kindly check

**Comment by Rakesh9100** [comment]
[Check here](https://github.com/Rakesh9100/Beautiify/pull/292/files), many extra main files are deleted @Prince-kushwaha-2004

**Comment by Rakesh9100** [comment]
I have fixed everything this time but take care from the next time @Prince-kushwaha-2004 

Merged 🥳 @Prince-kushwaha-2004

**Pull request merged**